browser/browsing_data/BUILD.gn

```diff
@@ -1,22 +1,9 @@
 source_set("browsing_data") {
-  # Remove when https://github.com/brave/brave-browser/issues/10657 is resolved
-  check_includes = false
   sources = [
-    "brave_browsing_data_remover_delegate.cc",
-    "brave_browsing_data_remover_delegate.h",
-    "brave_clear_browsing_data.cc",
     "brave_clear_browsing_data.h",
```

Comment:
I keep brave_clear_browsing_data.h file here because brave_browser_main_parts.cc has been including the file.  So, now the include chain is that //chrome/browser(with some //brave/browser/browsing_data/* files) -> //brave/browser -> //brave/browser/browsing_data/brave_clear_browsing_data.h. But, I'm not fully sure whether Brave allows to keep only header file in BUILD.gn.